src/main/g8/src/main/scala/$package__packaged$/JokeAlg.scala

```diff
@@ -0,0 +1,48 @@
+package $package$
+
+import cats.Applicative
+import cats.effect.Sync
+import cats.implicits._
+import io.circe.{Encoder, Decoder, Json, HCursor}
+import org.http4s.{EntityDecoder, EntityEncoder, Method, Uri, Request}
+import org.http4s.client.Client
+import org.http4s.circe._
+
+trait JokeAlg[F[_]]{
```

Comment:
Do we need `Alg` there, as opposed to just:

```scala
trait Jokes[F[_]] {
  def get: F[Joke]
}
```

`Alg` prefix reminds me of ugly java/C# practices (`IThing, `ThingImpl`), adds no info (apart from a tiny hint of Hungarian notation), and I would also posit that a good chunk of users would not know that `Alg` stands for algebra, and what algebra means.